crates/chain/tests/test_local_chain.rs

```diff
@@ -9,6 +9,7 @@ use bdk_chain::{
     },
     BlockId,
 };
+use bdk_testenv::{chain_update, h, local_chain};
```

Comment:
As we are importing this now from bdk_testenv, and deleted the macros from `commond/mod.rs` I guess we can remove the code at L16-17.

``` rust
#[macro_use]
mod common;
```